web/app/css/styles.css

```diff
@@ -2,18 +2,17 @@
 
 :root {
   --white: #fff;
-  --buttonblue: #4076c4;
-  --messageblue: #2F80ED;
+  --indigoblue: #4076c4;
+  --royalblue: #2F80ED;
   --darkblue: #071E3C;
+  --curiousblue: #2D9CDB;
+  --pictonblue: #56CCF2;
   --warmgrey: #f6f6f6;
   --coldgrey: #c9c9c9;
-  --latency-p99: var(--messageblue);
-  --latency-p95: #2D9CDB;
-  --latency-p50: #56CCF2;
-  --subheader-grey: #828282;
```

Comment:
there are some more references to `--subheader-grey` in the codebase that should also be updated